config/ftbquests/quests/chapters/mekanism.snbt

```diff
@@ -1393,7 +1394,7 @@
 			icon: "mekanism:ultimate_chemical_tank"
 			x: 7.5d
 			y: 9.0d
-			description: ["Oh! You're still here?"]
+			description: ["Oh! You're still here? Have some cookies for your determination!"]
```

Comment:
The 'have a cookie' was implied by the reward ^_~ No need to state it.

Reply:
Right :D